chaoscenter/web/config/oats.config.ts

```diff
@@ -9,7 +9,7 @@ function normalizePath(url: string): string {
 export default defineConfig({
   services: {
     auth: {
-      url: 'https://raw.githubusercontent.com/litmuschaos/litmus/master/mkdocs/docs/auth/v3.0.0/auth-api.json',
+      file: '../../mkdocs/docs/auth/v3.0.0/auth-api.json',
```

Comment:
Before merging the PR revert this file back

Reply:
@hrishavjha, I'd like to provide some context regarding the change to use a relative path in the config. The aim here is to ensure that code generation aligns with the most current OpenAPI specification, rather than being tied to a specific version of the Git repository. By referencing a local path, in future we can do things like having pre-commit checks to verify that the generated code remains consistent and hasn't been altered manually by mistake. Additionally, this approach prevents the scenario where executing `npx oats import --config config/oats.config.ts --verbose` after merging the PR might inadvertently revert the changes. I am happy to revert it if you have strong opinion on this but I would like to better understand your rational.